setup.py

```diff
@@ -0,0 +1,52 @@
+
+import os
+import glob
+import platform
+from setuptools import Extension, setup
+from setuptools.command.build import build
+
+class Build(build):
+    '''Custom build for setuptools to compile C++ shared libraries.'''
+
+    cpp_modules = ['ProbeParticle', 'GridUtils', 'fitSpline', 'fitting']
+
+    def run(self):
+        if platform.system() == 'Windows':
+            self.build_windows()
+        else:
+            self.make()
+        super().run()
+
+    def make(self):
+        package_path = os.path.dirname(os.path.realpath(__file__))
+        cpp_path = os.path.normpath(package_path + '/ppafm/cpp/')
+        current_directory = os.getcwd()
```

Comment:
I think it would be better to define those as a part of `__init__()` method, since you reuse them also in `build_windows`

Reply:
Done